### PR TITLE
Bump minimum supported chrome version from 32 to 33

### PR DESCRIPTION
**Pull request by sbc100**
This allows us to remove the promise polyfill: https://caniuse.com/promises

### CODE REVIEW

**Comment by kripken** [comment]
Where do you see that 33 is enough?

I can't find 33 on 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise

**Comment by sbc100** [comment]
https://caniuse.com/promises

**Comment by sbc100** [comment]
It looks like 32 had partial support.

File: ChangeLog.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ See docs/process.md for more on how version tagging works.
 
 3.1.74 (in development)
 -----------------------
+- The minimum supported chrome version was bumpd from 32 to 33. (#23077)
```
Comment (kripken) [inline ChangeLog.md:23]:
```suggestion
- The minimum supported chrome version was bumped from 32 to 33. (#23077)
```

**Review by kripken — APPROVED** [review]
lgtm. odd mdn&caniuse disagree, but it is safe to take the higher version i guess

**Pull request merged**